src/event/event_epoll.c

```diff
@@ -89,12 +89,12 @@ _dispatch_muxnote_bucket(int ident)
 	return &_dispatch_sources[DSL_HASH((uint32_t)ident)];
 }
 #define _dispatch_unote_muxnote_bucket(du) \
-	_dispatch_muxnote_bucket(du._du->du_ident)
+	_dispatch_muxnote_bucket((int)du._du->du_ident)
```

Comment:
I think _dispatch_muxnote_bucket() argument type should take a uint32_t instead.